src/tests/ftest/util/ServerUtils.py

```diff
@@ -36,22 +36,22 @@
 
 from avocado.utils import genio
 
-sessions = {}
+SESSIONS = {}
 
 class ServerFailed(Exception):
     """ Server didn't start/stop properly. """
 
 # a callback function used when there is cmd line I/O, not intended
 # to be used outside of this file
 def printFunc(thestring):
-        print("<SERVER>" + thestring)
+    print("<SERVER>" + thestring)
 
 def runServer(hostfile, setname, basepath, uri_path=None, env_dict=None):
     """
     Launches DAOS servers in accordance with the supplied hostfile.
 
     """
-    global sessions
+    global SESSIONS
```

Comment:
(pylint-global-variable-not-assigned) Using global for 'SESSIONS' but no assignment is done